lib/resolver.fallback.js

```diff
@@ -36,7 +36,7 @@ export default class PodletClientFallbackResolver {
      */
     // @ts-expect-error Deliberate default empty options for better error messages
     constructor(options = {}) {
-        this.#http = options.http || new HTTP();
+        this.#http = options.http || new HttpClient();
```

Comment:
Don't we need the same options here as in `resolver.content`?

Reply:
🤔 yeah, that does seem to make sense.